### PR TITLE
Add event-item and event-slot to Resurrect Event

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Adds an ItemStack and Slot event-values to the Entity Resurrect event. They return the item/slot used in the event (ie: the totem consumed). If no totem is present (and the event is therefore cancelled), the values will be null. Tested on both players and zombies.

Also removes the class existence checks because Skript now targets 1.13+.

---
**Target Minecraft Versions:** 
**Requirements:** 
**Related Issues:** https://github.com/SkriptLang/Skript/issues/5625

### CODE REVIEW

**Comment by Fusezion** [comment]
Related issue #5625

File: src/main/java/ch/njol/skript/events/SimpleEvents.java [hunk 1]
```diff
@@ -470,16 +470,14 @@ public class SimpleEvents {
...
-							"	uncancel the event")
-					.since("2.2-dev28");
-			SkriptEventHandler.listenCancelled.add(EntityResurrectEvent.class); // Listen this even when cancelled
-		}
+		Skript.registerEvent("Resurrect Attempt", SimpleEvent.class, EntityResurrectEvent.class, "[entity] resurrect[ion] [attempt]")
+				.description("Called when an entity dies, always. If they are not holding a totem, this is cancelled - you can, however, uncancel it.")
+				.examples("on resurrect attempt:",
+						"	entity is player",
+						"	entity has permission \"admin.undying\"",
+						"	uncancel the event")
```
Comment (AyhamAl-Ali) [inline src/main/java/ch/njol/skript/events/SimpleEvents.java:478]: While at it \:)
```suggestion
				.examples(
						"on resurrect attempt:",
						"\tentity is player",
						"\tentity has permission \"admin.undying\"",
						"\tuncancel the event"
				)
```

**Review by AyhamAl-Ali — APPROVED** [review]

File: src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java [hunk 2]
```diff
@@ -1562,5 +1564,37 @@ public Location get(LootGenerateEvent event) {
...
+					return null;
+
+				if (hand == EquipmentSlot.HAND) {
+					return equipment.getItemInMainHand();
+				} else if (hand == EquipmentSlot.OFF_HAND) {
+					return equipment.getItemInOffHand();
+				}
+				return null;
+			}
+		}, EventValues.TIME_NOW);
```
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java:1584]: You shouldn't need the itemtype when slot is present. The converters should do `event-item`

**Review by TheLimeGlass — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java (hunk 2 above)
Comment (sovdeeth) [reply src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java:1584]: oh cool i didn't realize it works like that!

**Review by Moderocky — APPROVED** [review]

**Review by TheLimeGlass — APPROVED** [review]

**Pull request merged**